### ISSUE TITLE
unused package?

### ISSUE THREAD

**Issue by jimmywarting** [ef4/ember-auto-import#402]
couldn't find/search for this in the code

https://github.com/ef4/ember-auto-import/blob/543fcc0dc2c9ffbff94f714aa0ad5b69e4fd54da/packages/ember-auto-import/package.json#L54

**Comment by ef4** [comment]
Yup, good catch, we dropped it at ed8583605482616f1217226b9010fbd54af0ad8d.

**Comment by jimmywarting** [comment]
https://github.com/ef4/ember-auto-import/blob/ed8583605482616f1217226b9010fbd54af0ad8d/ts/analyzer.ts#L3

FYI, You know that NodeJs have some newer methods in its toolchain that dose some of what fs-extra have had for years.
namely copy/remove/mkdir with recursive as an option. Just want to point that out
